docs/writing-tests/addon-vitest.mdx

```diff
@@ -0,0 +1,346 @@
+---
+title: 'Test runner with Vitest'
+sidebar:
+  order: 2
+  title: Test runner with Vitest
+---
+
+TK - Intro
+
+## Set up
+
+To get started, run the following command to install and configure the addon:
+
+{/* TODO: Snippetize */}
+```sh
+npx storybook@latest add @storybook/experimental-addon-vitest
+```
+
+That command will do the following:
+
+1. Install and register the Vitest addon, which contains the plugin to run your stories as tests
+1. Inspect your project's Vite and Vitest setup
+    1. If Vite is not installed and you're using the [`nextjs` framework](../get-started/frameworks/nextjs.mdx), it will install and configure Vite for you, as well as `vite-plugin-storybook-nextjs` (necessary to have your Next.js components function in Vitest).
+    1. Otherwise, if Vite is not installed, it will stop and point you to these instructions to continue setting it up in your project.
+    1. If Vite is installed, it will then check for Vitest.
+        1. If Vitest is not installed, it will:
+            1. Install `vitest`, `@vitest/browser`, and `playwright`
+            1. Run `npx playwright install chromium` to install the Chromium browser engine
+            1. Create a Vitest config file (`vitest.config.ts`) and a Vitest setup file (`storybook.setup.ts`)
+        1. If Vitest is installed, it will stop and point you to these instructions to continue setting it up in your project.
```

Comment:
It feels to me that we should just mention that the command will set up the plugin for the user. Hopefully the Storybook add command should be the place where the information is explained to the user as things are set up.

Then, we would have a section for manual steps that explains what to do:
- Make sure Vite and Vitest are configured and browser mode is used (point to vitest docs https://vitest.dev/guide/browser/)
- Install Next.js plugin if applicable (point to the docs https://github.com/storybookjs/vite-plugin-storybook-nextjs)
- Write set up file (point to portable stories docs)
- Write vitest config file

When it comes to the `npx playwright install chromium` command, Vitest doesn't mention it, I assume it's because it either does it for the user, or the user will get an error message from playwright telling them to run that using the correct package manager command. We should probably skip that in the docs

Feel free to use Vitest browser docs as reference:
https://vitest.dev/guide/browser/